scripts/fix_diffs.py

```diff
@@ -94,11 +94,17 @@ def main(fp):
 
 if __name__ == '__main__':
     import sys
+
     if len(sys.argv) < 2:
-        print('Usage: python fix_diffs.py <po_file_path>')
+        print('Usage: python fix_diffs.py <po_file_path> <original_po_file_commit_hash>')
```

Comment:
We should mark this argument as optional through marking the variable with a pair of square brackets or something similar. And also it would be great to show the default value (which is HEAD) here in some way.

Also I'd suggest to not use the word "hash" here as it can be HEAD, HEAD~3 etc. which are not hashes but represents a commit.

Reply:
Thanks for your suggestion! I've fixed it. Please check it whether proper.